js/output/pjs/pjs-output.js

```diff
@@ -66,8 +66,17 @@ window.PJSOutput = Backbone.View.extend({
         this.render();
         this.bind();
 
+        iframeOverlay.createRelay(this.$canvas[0]);
+
         this.build(this.$canvas[0]);
 
+        if (this.config.useDebugger && PJSDebugger) {
+            this.debugger = new PJSDebugger({
+                context: this.canvas,
+                output: this
+            });
+        }
```

Comment:
I think that's probably fine, and likely unavoidable. One gotchya here I think you'll need to use `window.PJSDebugger` to avoid the undeclared variable error.